src/file_pipe_log/log_file.rs

```diff
@@ -53,7 +132,8 @@ impl<F: FileSystem> LogFileWriter<F> {
         self.last_sync = 0;
         self.written = 0;
         let mut buf = Vec::with_capacity(LogFileHeader::len());
-        LogFileHeader::default().encode(&mut buf)?;
+        // LogFileHeader::default().encode(&mut buf)?;
```

Comment:
I feel like you can do a self-review next time to prune off these unneeded comments (it's easier to see them in GitHub PR view).

Reply:
Sure, it's a confusing and unnecessary annotation.